packages/api/src/modules/suggestion/model.ts

```diff
@@ -0,0 +1,54 @@
+import * as mongoose from 'mongoose';
+
+import { LANGS_LIST, ERROR_CODES } from './constants';
+
+const { Types } = mongoose.Schema;
+
+const ratingSchema = new mongoose.Schema({
+  positiveVotes: [Types.ObjectId],
+  negativeVotes: [Types.ObjectId],
+});
+
+const suggestionSchema = new mongoose.Schema(
+  {
+    phrase: {
+      type: Types.ObjectId,
+      ref: 'phrase',
+      required: true,
+      index: true,
+    },
+    user: {
+      type: Types.ObjectId,
+      ref: 'user',
+      required: true,
+      index: true,
+    },
+    lang: {
+      type: String,
+      enum: LANGS_LIST,
+      required: true,
+      index: true,
+    },
+    text: {
+      type: String,
+      required: true,
+      minlength: [1, ERROR_CODES.TEXT_SHORT],
+      maxlength: [10000, ERROR_CODES.TEXT_LONG],
+    },
+    rating: {
+      type: ratingSchema,
+      required: true,
+    },
+    approved: {
```

Comment:
não é melhor status, não? Uma suggestion vai ter só o status approved? A gente pode querer implementar um no futuro como abuse, spam, ou pending....

Reply:
Futuramente a gente pode pensar em implementar um campo "tags", aí acho que faz mais sentido. Por enquanto, pensando que uma frase vai ter várias sugestões e só uma única vai ser aprovada, acho que vai economizar espaço no banco de dados se houver um campo apenas opcional, que só vai aparecer se a suggestion for aprovada.